node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -144,7 +162,7 @@ func (r OsDeviceConnectivityHelperScsiGeneric) GetMpathDevice(volumeId string, l
 
 	for _, arrayIdentifier := range arrayIdentifiers {
 		dp := strings.Join([]string{targetPath, connectivityType, arrayIdentifier, "lun", lunIdStr}, "-")
-		logger.Infof("GetMpathDevice: Get the mpath devices related to connectivityType=%s initiator=%s and lunID=%s : {%v}", connectivityType, arrayIdentifier, lunIdStr, dp)
+		logger.Infof("GetMpathDevice: Get the mpath devices related to connectivityType=%s initiator=%s and lunID=%d : {%v}", connectivityType, arrayIdentifier, lunId, dp)
```

Comment:
I think the log should still include somewhere the value after conversion (lunIdStr)

Reply:
yes, last params (dp) has it

Reply:
I would like both before and after. (rawLunID=%s lunID=%s)

Reply:
ok